hr_timesheet_name_customer/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2015-17 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3.0).
+
+{
+    "name": "Name Customer",
+    "summary": "Add Description Customer",
```

Comment:
Make description more clear for end user. Eg
“Add ‘Description Customer’ field for timesheets”